BossMod/Modules/Shadowbringers/Foray/CriticalEngagement/CE12BayingOfHounds.cs

```diff
@@ -47,61 +60,26 @@ class ScorchingLash : Components.SelfTargetedAOEs
     {
         public ScorchingLash() : base(ActionID.MakeSpell(AID.ScorchingLash), new AOEShapeRect(50, 5)) { }
     }
-
-    class Hellpounce : Components.GenericAOEs
+    class Hellpounce : Components.ChargeAOEs
     {
-        private AOEInstance? _charge;
-
-        public Hellpounce() : base(ActionID.MakeSpell(AID.Hellpounce), "GTFO from charge!") { }
-
-        public override IEnumerable<AOEInstance> ActiveAOEs(BossModule module, int slot, Actor actor)
-        {
-            if (_charge != null)
-                yield return _charge.Value;
-        }
-
-        public override void OnCastStarted(BossModule module, Actor caster, ActorCastInfo spell)
-        {
-            if ((AID)spell.Action.ID is AID.Hellpounce or AID.HellpounceSecond)
-                Activate(caster.Position, spell.LocXZ, spell.FinishAt);
-        }
-
-        public override void OnCastFinished(BossModule module, Actor caster, ActorCastInfo spell)
-        {
-            switch ((AID)spell.Action.ID)
-            {
-                case AID.Hellpounce:
-                    var offset = spell.LocXZ - module.Bounds.Center;
-                    Activate(spell.LocXZ, module.Bounds.Center - offset, module.WorldState.CurrentTime.AddSeconds(3.7f));
-                    break;
-                case AID.HellpounceSecond:
-                    _charge = null;
-                    break;
-            }
-        }
-
-        private void Activate(WPos source, WPos target, DateTime activation)
-        {
-            var shape = new AOEShapeRect(0, 5);
-            shape.SetEndPoint(target, source, new());
-            _charge = new(shape, source, activation: activation);
-        }
+        public Hellpounce() : base(ActionID.MakeSpell(AID.Hellpounce), 5) { }
+    }
+    class HellpounceSecond : Components.ChargeAOEs
```

Comment:
Ok, so the current implementation would show the second hint for a very short time - this was the reason it was implemented as a custom component before.